css/front-legacy.less

```diff
@@ -3,6 +3,26 @@
 .panel-grid {
 	zoom: 1;
 	.clearfix();
+
+	.so-parallax {
+		position: relative;
+
+		& > div:not(.simpleParallax) {
+			position: relative;
+			z-index: 1;
+		}
+
+		.simpleParallax,
+		img[data-siteorigin-parallax] {
```

Comment:
Would you like to order selectors alphabetically for new rules as we do in themes?